grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -240,7 +240,8 @@ private void doClose(Status status, Metadata metadata, @Nullable Throwable excep
             return;
         }
 
-        checkState(!closeCalled, "call already closed");
+        checkState(!closeCalled, "call already closed. status: %s, metadata: %s, exception: %s",
+                   status, metadata, exception);
```

Comment:
Would it be OK to expose metadata here? (access token, etc)

Reply:
Good point.

Reply:
Fix at [`7f434a7` (#5061)](https://github.com/line/armeria/pull/5061/commits/7f434a78a7797b8437e8cf26998ad673d00ad45e)